wrapanapi/systems/rhevm.py

```diff
@@ -431,8 +431,8 @@ def _is_disk_ok(self, disk_id):
                 if self.system.api.follow_link(disk_attach.disk).id == disk_id].pop()
         return getattr(disk, 'status', None) == types.DiskStatus.OK
 
-    def add_disk(self, storage_domain=None, size=None, interface='VIRTIO', format=None,
-                 active=True):
+    def add_disk(self, storage_domain=None, size=None, interface='virtio', format='cow',
```

Comment:
why did we change case on `virtio`?

Reply:
`types.DiskInterface()` takes a lowercase string for the enum lookup. Defaulting to the case it expects, and explicitly passing a lowercase string within the lookup.